news/index.html

```diff
@@ -98,6 +98,12 @@ <h2> News </h2>
                 <!-- TODO: Remove duplication by making this a list of links to the articles -->
                 <div class="floe-news-archive">
                     <ul>
+                        <li>
+                            <a href="2018-04-18-New-Content-Added-to-Inclusive-Design-Resources">
```

Comment:
This link '2018-04-18-New-Content-Added-to-Inclusive-Design-Resources' is also missing the '.html' file extension.